dashboard/config/routes.rb

```diff
@@ -471,6 +470,7 @@
       get 'pull-review', to: 'peer_reviews#pull_review', as: 'pull_review'
     end
 
+    resources :courses, only: [:index]
```

Comment:
instead of adding a new `resources :courses`, please add something to the existing block below, such as:
```
  collection do
    get 'all'
  end
```